remote-method/index.js

```diff
@@ -0,0 +1,244 @@
+'use strict';
+var yeoman = require('yeoman-generator');
+var chalk = require('chalk');
+
+var workspace = require('loopback-workspace');
+var ModelProperty = workspace.models.ModelProperty;
+
+var actions = require('../lib/actions');
+var helpers = require('../lib/helpers');
+var validateName = helpers.validateName;
+var accepts = [];
+var returns = [];
+
+module.exports = yeoman.generators.Base.extend({
+  // NOTE(bajtos)
+  // This generator does not track file changes via yeoman,
+  // as loopback-workspace is editing (modifying) files when
+  // saving project changes.
+
+  help: function() {
+    return helpers.customHelp(this);
+  },
+
+  loadProject: actions.loadProject,
+
+  loadModels: actions.loadModels,
+
+  askForModel: function() {
+    if (this.options.modelName) {
+      this.modelName = this.options.modelName;
```

Comment:
I think you need to declare `options.modelName` in the constructor function to have it described in help. Could you please check?

See https://github.com/strongloop/generator-loopback/blob/99c66a66d4d1e1179749d792ac3c298fd825636c/app/index.js#L20-L29

BTW I think it makes more sense to make the name an argument. Below is the difference between options and arguments as I understand them (please check it yourself, I may be wrong!):

```
// argument
slc loopback:remote-method MyModel
// option
slc loopback:remote-method --modelName MyModel
```